### ISSUE TITLE
PEP 541 Request: krb5

### ISSUE THREAD

**Issue by jborean93** [pypa/pypi-support#1269]
### Project to be claimed

`krb5`: https://pypi.org/project/krb5

### Your PyPI username

`jborean93`: https://pypi.org/user/jborean93

### Reasons for the request

I went to publish https://github.com/jborean93/pykrb5 but PyPI errors with

>  The name 'krb5' isn't allowed. See https://pypi.org/help/#project-name for more information.

It's hard to tell whether this is reserved by PyPI itself or some other person/org. If it isn't possible to reserve this project name then it would be great to determine why and whether I could get in contact with someone who it is reserved under.

### Maintenance or replacement?

Replacement

### Source code repositories URLs

https://github.com/jborean93/pykrb5

### Contact and additional research

I tried finding a way to determine the owner of this name in PyPI but there are no project under this name and https://pypi.org/project/krb5 leads nowhere. Maybe it is reserved by https://pypi.org/project/python-krb5/#files but that hasn't been in updated since 2009 and it's no immediately apparent it actually uses the `krb5` name. The project name is `python-krb5` and it has an extension called `krb5` but I'm unsure if that extension is what is causing the problem here.

```python
from distutils.core import setup, Extension

setup	(name = 'python-krb5',
	version = '0.2',
	description = 'Kerberos 5 Bindings for Python',
        long_description = """
This extension all python programs to access the MIT Kerberos 5 libraries to manipulate principals and credentials.
""",
	author = 'Benjamin Montgomery',
	author_email = 'bmontgom@montynet.org',
	classifiers = [
		'Development Status :: 4 - Beta',
		'License :: OSI Approved :: GNU General Public License (GPL)',
		'Operating System :: POSIX :: Linux',
		'Programming Language :: C',
		'Topic :: System :: Systems Administration :: Authentication/Directory'
		],
	ext_modules = 	[Extension	('krb5', 
					['python-krb5.c', 'principal.c', 'credential.c', 'error.c'],
					libraries = ['krb5', 'com_err']
					)
			]
	)
```

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

**Comment by webknjaz** [comment]
Hey @yeraydiazdiaz, could you help with this if you have a minute?

**Comment by yeraydiazdiaz** [comment]
Hi @jborean93, the project name `krb5` was placed in the prohibited project names list, @pypa/warehouse-admins could you release the name, please?

**Comment by jborean93** [comment]
Thanks @yeraydiazdiaz for the response, is this something that's up to the warehouse-admins to either approve will I have to use a different name?

**Comment by webknjaz** [comment]
Only the admins have the power to manage the blocklist.

**Comment by ewdurbin** [comment]
Project released to the `jborean93` User on PyPI.

**Comment by jborean93** [comment]
Thanks everyone!

**Comment by jborean93** [comment]
Sorry to revive the dead issue but am I doing anything wrong? When going to upload the release the wheels work fine but the sdist has failed https://github.com/jborean93/pykrb5/runs/3591385739?check_suite_focus=true

>  This filename has already been used, use a different version. See https://pypi.org/help/#file-name-reuse for more information.

I see the wheels are available on https://pypi.org/project/krb5/#files but the sdist is definitely not there. The publish action is quite simple AFAIK https://github.com/jborean93/pykrb5/blob/main/.github/workflows/ci.yml#L167-L172.

**Comment by ewdurbin** [comment]
The release version `0.1.0` was previously used when registered before, so the file `krb5-0.1.0.tar.gz` cannot be reused.

**Comment by ewdurbin** [comment]
So in your case, wheels were published just fine since _only_ `krb5-0.1.0.tar.gz` was uploaded at that time.

You can increment your version to release all your artifacts.

**Comment by jborean93** [comment]
Thank you very much, `0.1.1` was successfully published to PyPI now.
